### ISSUE TITLE
Improve how the submission status messages are made and displayed

### ISSUE THREAD

**Issue by ThunderAxe31** [TASVideos/tasvideos#676]
Currently we're still using a system in which the judgements and other submission updates are written at the bottom of the submission text. This approach is too much hacky and lame. We need to move to a much cleaner approach.

1. Make a separate space for the submission status messages, so that we the submission text is used only for describing the movie.
2. When someone attempts to change the status of submission, the site should spawn a textbox in which the user can put a comment, for which the form will give a default text for people with no creativity ("Judging." "Cancelling." etc).
3. Even if the user doesn't enter a comment, the site should still denote the event by date, user, and what's the submission state changed.
4. The comment should be editable by its author and by moderators, but the date, user, and state shouldn't be editable or removable.

Have a mockup screenshot of my idea!

![coolstuff](https://user-images.githubusercontent.com/36802254/146361333-b8be822c-220b-44e9-a675-df87fb9e2c8d.png)

**Comment by meshuggahtas** [comment]
I think it's mostly OK. My only personal minor opinion would be not repeating usernames between status and their own message.

Example:
2021/12/31, 11:43 feos published the movie.
It was even worse than what I predicted, but it was worth it! Publishing.

**Comment by ThunderAxe31** [comment]
Claiming for publishing and actually publishing should be considered as two different status change. This isn't currently supported, and in my opinion it should be introduced.

**Comment by ThunderAxe31** [comment]
> I think it's mostly OK. My only personal minor opinion would be not repeating usernames between status and their own message.
> 
> Example: 2021/12/31, 11:43 feos published the movie. It was even worse than what I predicted, but it was worth it! Publishing.

I don't think that's a good idea. Sometimes people may not having anything to say and leave just the status update without a message, while some other times people may wanting to say something important but not in conjunction with an action, which is something that already happened in the past, for example when a judge gave an additional explanation post-judgement or when an important information needed to be noted at some point.

**Comment by vadosnaprimer** [comment]
Yeah since it's just a log, putting username on every entry is better. If one wants to write several paragraphs in one entry, it's also fine. Let the user decide how to split the messages.
